changelogs/fragments/4320-nmcli-hairpin.yml

```
@@ -0,0 +1,2 @@
+breaking_changes:
+  - nmcli - deprecate default hairpin mode for a bridge. This so we can change it to 'no' in ansible 6.0.0, as this is also the default in nmcli.
```

Since 5.0.0 is about to be released very soon, we should only change the default in 7.0.0 (see https://github.com/ansible-collections/community.general/issues/582):
```suggestion
  - nmcli - deprecate default hairpin mode for a bridge. This so we can change it to ``no`` in community.general 7.0.0, as this is also the default in nmcli.
```